test/expected.txt

```diff
@@ -17,6 +17,8 @@
 || find: Finds all persons whose names contain any of the specified keywords (case-sensitive) and displays them as a list with index numbers.
 || Parameters: KEYWORD [MORE_KEYWORDS]...
 || Example: find alice bob charlie
+|| sort: Sorts all persons in the address book.
```

Comment:
Possible to include more details on how the sorting works? See "find" description above at line 17, it explains the list with index numbers and also specifies conditions such as (case-sensitive).